.github/workflows/dist-docs.yml

```diff
@@ -0,0 +1,127 @@
+name: Rebuild src/static/support/dist-docs
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build-linux:
+    env:
+      dependencies: |
+        automake libtool gcc bc libjemalloc1 libjemalloc-dev    \
+        libssl-dev llvm-dev libelf-dev libnuma-dev libpcap-dev  \
+        selinux-policy-dev \
+        ghostscript markdown
+      m32_dependecies: gcc-multilib
+      CC:          ${{ matrix.compiler }}
+      LIBS:        ${{ matrix.libs }}
+      M32:         ${{ matrix.m32 }}
+      OPTS:        ${{ matrix.opts }}
+      TESTSUITE:   ${{ matrix.testsuite }}
+      ASAN:        ${{ matrix.asan }}
+
+    name: linux ${{ join(matrix.*, ' ') }}
+    runs-on: ubuntu-18.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - compiler:     gcc
+            opts:         --disable-ssl
+
+    steps:
+    - name: checkout ovn-website
+      uses: actions/checkout@v2
+      with:
+        path: 'ovn-website'
+        ref: 'master'
+
+    - name: checkout OVN master
+      uses: actions/checkout@v2
+      with:
+        repository: 'ovn-org/ovn'
+        path: 'ovn'
+        ref: 'master'
+        submodules: recursive
+
+    - name: update APT cache
+      run:  sudo apt update
+
+    - name: install required dependencies
+      run:  sudo apt install -y ${{ env.dependencies }}
+
+    - name: install libunbound libunwind
+      if:   matrix.m32 == ''
+      run:  sudo apt install -y libunbound-dev libunwind-dev
+
+    - name: install 32-bit dependencies
+      if:   matrix.m32 != ''
+      run:  sudo apt install -y ${{ env.m32_dependecies }}
+
+    - name: update PATH
+      run:  |
+        echo "$HOME/bin"        >> $GITHUB_PATH
+        echo "$HOME/.local/bin" >> $GITHUB_PATH
+
+    - name: set up python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: prepare ovn
+      run:  |
+        pushd ./ovn
+        ./.ci/linux-prepare.sh
+        popd
+
+    - name: build ovn
+      run:  |
+        pushd ./ovn
+        ./.ci/linux-build.sh
+        popd
+
+    - name: build ovn dist-docs
+      run:  |
+        pushd ./ovn
+        make dist-docs
+        popd
+
+    - name: replace dist-docs content
+      if: ${{ success() }}
+      run:  |
+        pushd ./ovn
+        tar -czvf /tmp/dist-docs.tgz dist-docs
+        popd
+        pushd ./ovn-website
+        pushd ./src/static/support
+        rm -rf ./dist-docs
+        tar xzvf  /tmp/dist-docs.tgz
```

Comment:
> It seems odd to create the tarball only to untar it a few lines down. Would a `cp -r` do the same thing without the extra overhead?
> 
> Even if you want to tar and untar, I don't see a good reason to use the 'z' option for gzip compression.

the tar idea was related to the archiving that happens below. I always use 'z' because I'm paranoid about disk space.
I will update this pr so the tar is part of the artifact and maybe that will make more sense. Please do tell me what you think about that and we will take the next step.